crypto/schemes.go

```diff
@@ -197,7 +194,46 @@ func NewPedersenBLSUnchainedSwapped() (cs *Scheme) {
 		ThresholdScheme: ThresholdScheme,
 		AuthScheme:      AuthScheme,
 		DKGAuthScheme:   DKGAuthScheme,
-		Pairing:         Pairing,
+		IdentityHash:    IdentityHashFunc,
+		DigestBeacon:    DigestFunc,
+	}
+}
+
+// SigsOnG1ID is the scheme id used to set unchained randomness on beacons with signatures on G1 that are
+// compliant with the hash to curve RFC.
+const SigsOnG1ID = "bls-unchained-g1-rfc"
```

Comment:
If you name "RFC" then best go along with naming the RFC version number like `rfcv16`?

Reply:
[RFC9380](https://www.rfc-editor.org/auth48/rfc9380)